evap/results/tests/test_views.py

```diff
@@ -14,13 +19,78 @@
 import random
 
 
-class TestResultsView(WebTestWith200Check):
+class TestResultsView(WebTest):
     url = '/results/'
-    test_users = ['manager']
 
-    @classmethod
-    def setUpTestData(cls):
-        mommy.make(UserProfile, username='manager', email="manager@institution.example.com")
+    @patch('evap.evaluation.models.Evaluation.can_be_seen_by', new=lambda self, user: True)
```

Comment:
I found that lambda part really hard to parse. The comma looks like it's separating the `new` argument from the next argument (which I thought was called `user`). Can we put parentheses around the `self, user` or do something else to improve this?